janus_server/src/datastore.rs

```diff
@@ -65,6 +65,7 @@ impl Datastore {
         }
     }
 
+    #[tracing::instrument(skip(self, f))]
```

Comment:
I think we should add `err` alongside `skip` so that we emit events when these functions return errors.